targets/javascript/templates/PlayFab_Api.js.ejs

```diff
@@ -60,21 +60,25 @@ if(!PlayFab._internalSettings) {
             var startTime = new Date();
             var requestBody = JSON.stringify(request);
 
-            var completeUrl = url;
+            var urlArr = ["", url]; //make a new array for the URL
```

Comment:
You don't need to init with an empty string here.
It can just be [url] as a list of one.